example-workflows/churn-ensemble.md

```diff
@@ -11,8 +11,8 @@ The philosophy behind the Aqueduct SDK is that you should be able to connect to
 **Outline:**
```

Comment:
@eunice-chan, this notebook is actually autogenerated. @jegonzal has a script that he wrote to convert our example notebooks into Markdown. Could you please use that script to update this page?

Reply:
I can do that--where can I find this script?